docs/source/conf.py

```diff
@@ -245,8 +245,8 @@
     (
         "index",
         "ReadtheDocsTemplate",
-        u"Read the Docs Template Documentation",
-        u"Read the Docs",
+        "Read the Docs Template Documentation",
+        "Read the Docs",
         "ReadtheDocsTemplate",
         "One line description of project.",
```

Comment:
This needs updating to actual information while you're making changes here, @cortadocodes

Reply:
I've removed this section now as we don't use it.